test/unittests/statetest_loader_test.cpp

```diff
@@ -0,0 +1,72 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <test/statetest/statetest.hpp>
+
+using namespace evmone;
+
+TEST(statetest_loader, load_empty_test)
+{
+    std::istringstream s{"{}"};
+    EXPECT_THROW(test::load_state_test(s), std::invalid_argument);
+}
+
+TEST(statetest_loader, load_minimal_test)
+{
+    std::istringstream s{R"({
+        "test": {
+            "_info": {},
+            "pre": {},
+            "transaction": {
+                "gasPrice": "",
+                "sender": "",
+                "to": "",
+                "data": null,
+                "gasLimit": "0",
+                "value": null
+            },
+            "post": {},
+            "env": {
+                "currentNumber": "0",
+                "currentTimestamp": "0",
+                "currentGasLimit": "0",
+                "currentCoinbase": ""
+            }
+        }
+    })"};
+
+    /*
+    struct StateTransitionTest
+    {
+        struct Case
+        {
+            struct Expectation
+            {
+                TestMultiTransaction::Indexes indexes;
+                hash256 state_hash;
+                hash256 logs_hash;
+                bool exception = false;
+            };
+
+            evmc_revision rev;
+            std::vector<Expectation> expectations;
+        };
+
+        state::State pre_state;
+        state::BlockInfo block;
+        TestMultiTransaction multi_tx;
+        std::vector<Case> cases;
+        std::unordered_map<uint64_t, std::string> input_labels;
+    };
+    */
+
+    const test::StateTransitionTest st = test::load_state_test(s);
+    // EXPECT_EQ(st.pre_state, {});
+    EXPECT_EQ(st.pre_state.get_accounts().size(), 0);
```

Comment:
Are you going to add get_accounts() const to fix the build?

Reply:
Done.